desktop/core/src/desktop/js/reactComponents/FileChooser/FileChooserModal/FileChooserModal.tsx

```diff
@@ -89,10 +109,23 @@ const FileChooserModal: React.FC<FileProps> = ({ show, onCancel, title, okText }
       onCancel={handleCancel}
       okText={okText}
       width={930}
-      className="file-chooser__modal"
+      className="hue-file-chooser__modal"
     >
-      <Spin spinning={loading}>
-        <Menu items={fileSystemList} onSelect={onPathChange} className="file-system__panel"></Menu>
+      <Spin spinning={loading} wrapperClassName="hue-file-chooser-modal__spinner">
```

Comment:
As mentioned above, check if we can modify the spinner using the component parameter instead of using the wrapper className